cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,25 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io/#/login');
+  });
+
   it('should provide an ability to log in', () => {
+    cy.url()
+      .should('include', '/login');
+
+    cy.get('h1.text-xs-center')
+      .should('contain.text', 'Sign In');
+
+    cy.get(':nth-child(1) > .form-control')
```

Comment:
Avoid using CSS selectors that are not specific or may change over time. This makes your test more brittle. Try to use id or data-test attributes.